lib/cosmos/tools/tlm_grapher/tabbed_plots_tool/tabbed_plots_realtime_thread.rb

```diff
@@ -19,7 +19,7 @@ class TabbedPlotsRealtimeThread < InterfaceThread
 
     # Create a new TabbedPlotsRealtimeThread
     def initialize(tabbed_plots_config, connection_success_callback = nil, connection_failed_callback = nil, connection_lost_callback = nil, fatal_exception_callback = nil)
-      interface = TcpipClientInterface.new('localhost', nil, System.ports['CTS_PREIDENTIFIED'], nil, tabbed_plots_config.cts_timeout, 'PREIDENTIFIED')
+      interface = TcpipClientInterface.new(System.connect_hosts['CTS_PREIDENTIFIED'], nil, System.ports['CTS_PREIDENTIFIED'], nil, tabbed_plots_config.cts_timeout, 'PREIDENTIFIED')
```

Comment:
This is the only place I see the new "connect_hosts" being used.  It seems like in order for the connect and the listen to link up, the connect should be used more places.  I guess I don't understand the CONOPs here.

Is the idea that the system.txt on a box with CTS would have listen address = 0.0.0.0, while the system.txt on some other machine that isn't running CTS would have connect address = <IP.OF.THE.BOXWITHCTS>?